OpenRA.Mods.Common/Traits/Palettes/PaletteFromGrayscale.cs

```diff
@@ -0,0 +1,58 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[TraitLocation(SystemActors.World | SystemActors.EditorWorld)]
+	[Desc("Creates a greyscale palette without any base palette file.")]
+	class PaletteFromGrayscaleInfo : TraitInfo
+	{
+		[PaletteDefinition]
+		[FieldLoader.Require]
+		[Desc("internal palette name")]
```

Comment:
```suggestion
		[Desc("Internal palette name")]
```

Reply:
Copied the palettefromrgba. Needs to be changed there too. Different pr?